src/input/event/value.rs

```diff
@@ -101,6 +113,11 @@ impl InputValue {
                             },
                             // Gamepad Button -> Keyboard
                             Capability::Keyboard(_) => Ok(self.clone()),
+                            Capability::Touchpad(touch) => match touch {
+                                Touchpad::LeftPad(_) => todo!(),
+                                Touchpad::RightPad(_) => todo!(),
+                                Touchpad::CenterPad(_) => todo!(),
```

Comment:
We should make sure to replace these `todo!()` with `Err(TranslationError::NotImplemented)` so the thread doesn't crash if someone tries to do this translation in an input profile.